### PR TITLE
Add fix_failed_payment mutation

### PR DESCRIPTION
**Pull request by ds300**
I tested against local exchange and it's working :+1:

~This should be merged after artsy/exchange#396 is in production~

### CODE REVIEW

File: _schema.graphql [hunk 1]
```diff
@@ -3477,6 +3459,20 @@ type FilterSaleArtworksCounts {
   ): FormattedNumber
 }
 
+input fixFailedPaymentInput {
```
Comment (ds300) [inline _schema.graphql]: why is this lower case?

**Comment by sweir27** [comment]
Can we add #donotmerge to this until the exchange-side is deployed to production? 🙏

File: _schema.graphql (hunk 1 above)
Comment (orta) [reply _schema.graphql]: interesting, so is type `fixFailedPaymentPayload`

**Comment by ds300** [comment]
Good catch! 🙇

File: _schema.graphql (hunk 1 above)
Comment (ds300) [reply _schema.graphql]: It was lowercase because I made a typo 😞

**Comment by ds300** [comment]
Alright I think this is 🍏 – As a bonus I've been doing end-to-end tests with force 👌

**Pull request merged**